x/mint/types/params.go

```diff
@@ -43,11 +43,11 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:           sdk.DefaultBondDenom,
-		InflationRateChange: sdk.NewDecWithPrec(13, 2),
-		InflationMax:        sdk.NewDecWithPrec(20, 2),
-		InflationMin:        sdk.NewDecWithPrec(7, 2),
+		InflationRateChange: sdk.ZeroDec(),
```

Comment:
What does this do?

Reply:
setting to zero doesn't allow for the inflation rate to change.

Reply:
Note that this was set to 100% or 1 on the hub not too long ago. Zaki has some charts for rationale here.